tox.ini

```diff
@@ -74,7 +74,7 @@ deps =
 description = Run unit tests
 
 [testenv:integration]
-commands = pytest -v --tb native --asyncio-mode=auto {[vars]tst_path}integration --log-cli-level=INFO -s {posargs}
+commands = pytest -v --tb native --asyncio-mode=auto {[vars]tst_path}integration --log-cli-level=INFO -s {posargs} --model kubeflow
```

Comment:
What would be the reason to add this here and remove it from the tox command in the workflow?

Reply:
Reason is above.

Reply:
I wrote the reason inside the [commit message](https://github.com/canonical/kubeflow-volumes-operator/pull/115/commits/733426d6d5a183441fefd5320f0e93fe7bd03f46). Not sure if you missed this or ask for a better explanation.

>Modify pytest command to always run tests on model kubeflow. Otherwise,
they fail deterministaclly since kubeflow-dashboard requires `kubeflow`
namespace.

Reply:
Sorry, I missed the commit message. In general we avoid setting this here in the pytest command, and rather set the model (therefore namespace) in the tox command in the GH workflow, so: `tox -e integration -- --model kubeflow`. Did this give you trouble?

Reply:
No, it's the `tox -e integration` without any arguments that I ran and failed after a while. I am OK with reverting this change in order to be consistent with what we do in our repos. My argument here is more that I do not see the point in having to pass the `--model kubeflow` args on every use of `tox integration`, when there is no other way to run the tests essentially.

Reply:
Let's add back the `--model kubeflow` as arg to the tox command, that way we keep consistency and also don't bury this option in case the namespace config changes in the future.

Reply:
Pushed a commit to revert the change!